Code_Exercises/CMakeLists.txt

```diff
@@ -9,28 +9,27 @@
 ]]
 
 # CMake function to build ab AdaptiveCpp configuration and create executable binaries
-# from AdaptiveCpp libraries.
 function( add_sycl_executable_adaptivecpp prefix source )
   set(TARGET_NAME "${prefix}_${source}")
   add_executable("${TARGET_NAME}" "${source}.cpp")
-  add_sycl_to_target(TARGET ${TARGET_NAME} SOURCES "${source}.cpp")
-
+  add_sycl_to_target(TARGET ${TARGET_NAME} SOURCES "${source}.cpp") 
+  
   target_include_directories("${prefix}_${source}" PRIVATE
-    ${PROJECT_SOURCE_DIR}/Utilities/include
+    ${PROJECT_SOURCE_DIR}/Utilities/include 
     ${PROJECT_SOURCE_DIR}/External/stb)
-  target_link_libraries("${prefix}_${source}" PRIVATE Threads::Threads)
+  target_link_libraries("${prefix}_${source}" PRIVATE Threads::Threads) 
   target_link_libraries("${prefix}_${source}" PUBLIC Catch2::Catch2)
-  set_target_properties("${prefix}_${source}" PROPERTIES CXX_STANDARD 17 CXX_STANDARD_REQUIRED ON)
+  set_target_properties("${prefix}_${source}" PROPERTIES CXX_STANDARD 17 CXX_STANDARD_REQUIRED ON)  
 endfunction()
 
 # CMake function to build a DPC++ configuration and create executable binaries
-# from oneAPI libraries.
+# from oneAPI libraries. 
 function( add_sycl_executable_dpcpp prefix source )
   set(TARGET_NAME "${prefix}_${source}")
-
+      
```

Comment:
A few lines in here add whitespace. Please change so there is no whitespace in the diff

Reply:
I think if you rebase on main, these changes should go away